kitsune/search/v2/signals/tests/test_forums.py

```diff
@@ -0,0 +1,40 @@
+from kitsune.search.v2.tests import Elastic7TestCase
+from kitsune.forums.tests import PostFactory
+from elasticsearch7.exceptions import NotFoundError
+
+from kitsune.search.v2.documents import ForumDocument
+
+
+class ForumDocumentSignalsTests(Elastic7TestCase):
+    def setUp(self):
+        self.post = PostFactory()
+        self.post_id = self.post.id
+
+    @property
+    def _doc(self):
```

Comment:
I would remove the underscore

Reply:
nit: I would also remove the property here. My preference would be for the tests to be as atomic as possible even if that means boilerplate code

Reply:
I could change this to a non-property `get_doc()` to make its usage in tests a bit clearer ("oh it gets the doc from ES fresh every time")